packages/angular/projects/clr-angular/src/utils/popover/providers/popover-toggle.service.ts

```diff
@@ -57,6 +58,8 @@ export class ClrPopoverToggleService {
    * This is for instance the case of components that open on a click, but close on a click outside.
    */
   public toggleWithEvent(event: any) {
+    preventArrowKeyScroll(event);
```

Comment:
This looks like the correct solution to the issue that was reported. 

For a unit test I would like to see a spec added to https://github.com/vmware/clarity/blob/master/packages/angular/projects/clr-angular/src/utils/popover/popover-close-button.spec.ts In the `Template API` suite. 

Something like 
```it('does not close the popover when called with an arrow key event', function (this: Context) { ...})```

Just create the specific arrow keypresses that the util function checks for and assert that the popover stays open. 

An example of creating a specific event for a spec and then testing something with it is here: https://github.com/vmware/clarity/blob/master/packages/angular/projects/clr-angular/src/utils/popover/providers/popover-events.service.spec.ts#L83